pom.xml

```diff
@@ -4,53 +4,151 @@
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.snomed.otf</groupId>
-	<artifactId>classification-service</artifactId>
-	<version>7.1.0-SNAPSHOT</version>
-
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
 		<version>2.1.3.RELEASE</version>
 	</parent>
 
+	<groupId>org.snomed.otf</groupId>
+	<artifactId>classification-service</artifactId>
+	<version>7.0.0</version>
+
```

Comment:
In develop it needs to be a SNAPSHOT version.

Reply:
Will do